dockstore-webservice/pom.xml

```diff
@@ -1290,7 +1290,7 @@
                     <!-- <inputSpec>https://raw.githubusercontent.com/dockstore/tool-registry-service-schemas/2.0.0-beta.5/src/main/resources/swagger/ga4gh-tool-discovery.yaml</inputSpec> -->
                     <!-- Change "serviceType.setVersion("2.0.1");" in ServiceInfoApiServiceImpl.java also -->
 		    <!-- note that the version of the TRS api in our fork is badly named and is not in sync with the main repo -->
-                    <inputSpec>https://raw.githubusercontent.com/dockstore/tool-registry-service-schemas/2.0.7-beta.1-notebooks/openapi/openapi.yaml</inputSpec>
+                    <inputSpec>https://raw.githubusercontent.com/dockstore/tool-registry-service-schemas/refs/heads/feature/more_paths/openapi/openapi.yaml</inputSpec>
```

Comment:
Should tool-registry-service-schemas#19 be merged first and tagged, then reference the tag from here?

Reply:
yup